pyrevolve/genotype/plasticoding/plasticoding.py

```diff
@@ -686,7 +723,13 @@ def __init__(self,
                  axiom_w=Alphabet.CORE_COMPONENT,
                  i_iterations=3,
                  max_structural_modules=100,
-                 robot_id=0
+                 robot_id=0,
+                 move_to_new=False,
+                 max_clauses=2,
+                 max_terms_clause=2,
+                 plastic=False,
+                 environmental_conditions=['hill'],
```

Comment:
If you want to add a new condition here, there is nowhere written that you also MUST create this variable inside the early_development function of plasticoding.py

And I don't like this way of doing things. You shouldn't touch multiple files all over the place when you need to configure one thing. And you shouldn't touch files outside of your experimental folder when configuring an experiment.